packages/xml-for-vscode/src/test/suite/extension.test.ts

```diff
@@ -0,0 +1,127 @@
+import * as path from "path";
+import { getDocUri, activate, sleep } from "./helper";
+import * as vscode from "vscode";
+import { promises as fs } from "fs";
+import { expect } from "chai";
+
+const SYNTAX_ERROR_MSG = "Syntax error";
+const docPath = path.resolve(
+  __dirname,
+  "..",
+  "..",
+  "..",
+  "src",
+  "test",
+  "testFixure",
+  "test.xml"
+);
+const docUri = getDocUri(docPath);
+
+describe("Extension Test", () => {
+  before(async function() {
```

Comment:
More **encapsulation** of tests desired.
Possibilities:
- add afterEach to clean the content? (I think this is better)
- open a different for each test?